settings.gradle.kts

```diff
@@ -1,2 +1,2 @@
 
-rootProject.name = "kmmresult"
\ No newline at end of file
+rootProject.name = "KmmResult"
```

Comment:
(needed to make the project work on non-case-sensitive OS)

Reply:
i don't know what that does to the publication, though